Explorer/Assets/Scripts/ECS/SceneLifeCycle/Components/VisualSceneState.cs

```diff
@@ -7,7 +7,9 @@ namespace ECS.SceneLifeCycle.Components
     public struct VisualSceneState
     {
         public VisualSceneStateEnum CurrentVisualSceneState;
+        public VisualSceneStateEnum CandidateVisualSceneState;
         public bool IsDirty;
+        public float TimeToChange;
```

Comment:
Please write documentation to provide some context about this new field, why is it necessary, what is its purpose or who uses it for what.